server/restapi/views.py

```diff
@@ -8,3 +9,36 @@ class UserView (generics.CreateAPIView):
 
 class MessageView(generics.CreateAPIView):
     serializer_class = MessageSerializer
+
+
+'''
+Запрос на вывод истории переписки (messageList)
+Пример запроса:
+{
+    "sender": "6bc2d415-f063-475a-96ee-56aabb9b9c60",
+    "recipient": "2c9a66c4-93ac-4e34-b972-99244ffc1743"
+}
+"sender" -- пользователя, делающего запрос
+"recipient" -- собеседника, переписка с которым запрашивается
+'''
+
+
+class MessageListView(generics.ListAPIView):
+    '''Возвращает клиенту все сообщения, которые адркссуются ему'''
+    serializer_class = MessageSerializer
+
+    def get_queryset(self):
+        queryset = Messages.objects.all()
+        # sender = self.request.query_params.get('sender', None)
+        recipient = self.request.query_params.get('recipient', None)
+        if recipient is not None:
+            queryset = queryset.filter(
+                # messages__sender=sender,
+                messages__recipient=recipient
```

Comment:
Я это исправил и фильтр заработал
```suggestion
                recipient_id=recipient
```